builder/image/templates/builder

```diff
@@ -101,7 +101,7 @@ if [ ! -f Dockerfile ]; then
     if [ -f /buildpacks ]; then
         BUILD_OPTS+=' -v /buildpacks:/tmp/buildpacks:rw'
         # give non-root slugbuilder user R/W perms for docker volumes
-        chown -R 2000:2000 /buildpacks
+        sudo chown -R 2000:2000 /buildpacks
```

Comment:
Can't we move this chown into the slugbuilder? https://github.com/deis/deis/blob/master/builder/image/slugbuilder/Dockerfile#L7